modules/virt-hot-plugging-memory.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-hot-plugging-memory_{context}"]
+
+= Hot plugging memory on a virtual machine
+
+You can add or remove the amount of memory allocated to a VM without having to restart the VM by using the {product-title} web console or editing the YAML file.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines*.
+. Select the required VM to open the *VirtualMachine details* page.
+. On the *Configuration* tab, click *Edit CPU|Memory*.
+. Enter the desired amount of memory and click *Save*.
+
+The system applies these changes immediately.

```

Comment:
i think it's worth adding that after change is applied live-migration is triggered.
in case VM is not migratable or spec change cannot be live-updated, `RestartRequired` condition will be added to VM